src/app/(home)/meeting/_components/AttendeeProfiles.tsx

```
@@ -0,0 +1,52 @@
+
+interface AttendeeProfilesInterface {
+	meetingId: number;
+	imgSize?: number;
+}
+
+const AttendeeProfiles = ({
+	meetingId,
+	imgSize = 28,
+}: AttendeeProfilesInterface) => {
+	const maxProfiles = 4;
+	const [profiles, setProfiles] = useState(['1', '2', '3', '4', '5', '6']);
```

ESLint

[error] 'setProfiles' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)